io/eolearn/tests/test_raster_io.py

```diff
@@ -360,3 +360,33 @@ def test_time_dependent_feature_with_timestamps(test_eopatch):
     new_eopatch = import_task(test_eopatch, filename=filename)
 
     assert_array_equal(new_eopatch[feature], test_eopatch[feature])
+
+
+@pytest.mark.parametrize(
+    "no_data_value, data_type", [(None, np.uint16), (np.nan, float), (0, int), (None, float), (1, np.byte)]
+)
+def test_export_import_sequence(no_data_value, data_type):
+    eopatch = EOPatch()
+    eopatch.bbox = BBox((0, 0, 1, 1), crs=CRS.WGS84)
+    feature = (FeatureType.DATA_TIMELESS, "DATA")
+
+    np_arr = np.zeros((10, 10, 1), dtype=data_type)
+    np_arr[:5, :5, :] = 1
```

Comment:
should this always be set to 1? Do the `np.nan` even play a role in the tests then?